libraries/AC_AttitudeControl/AC_PosControl_Sub.h

```diff
@@ -31,6 +31,9 @@ class AC_PosControl_Sub : public AC_PosControl {
     ///     set force_descend to true during landing to allow target to move low enough to slow the motors
     void set_alt_target_from_climb_rate_ff(float climb_rate_cms, float dt, bool force_descend) override;
 
+    /// relax_alt_hold_controllers - set all desired and targets to measured
+    void relax_alt_hold_controllers(float throttle_setting, bool reset_i = true);
```

Comment:
Please declare it `override`.

Reply:
We will also need to declare it `virtual` here: https://github.com/ArduPilot/ardupilot/blob/0b873f77788ce85cdfb243604d0f6cf24272f28e/libraries/AC_AttitudeControl/AC_PosControl.h#L124

Reply:
> Please declare it override.

Nevermind, I didn't notice the different signature when I suggested this.


We should give a compiler error if the original function from AC_PositionControl.h without the `bool reset_i` parameter is used.

So in `AC_PosControl_Sub.h` we should have:
```
 // override, but don't define this will give compiler error if attempted
 void relax_alt_hold_controllers(float throttle_setting) override;

 // don't use default argument in order not to shadow
 void relax_alt_hold_controllers(float throttle_setting, bool reset_i);
```

and in `AC_PosControl.h` we should have:
```cpp
 virtual void relax_alt_hold_controllers(float throttle_setting);
```